src/EventStore.Core/Certificates/OptionsCertificateProvider.cs

```diff
@@ -19,6 +19,10 @@ public class OptionsCertificateProvider: CertificateProvider {
 
 			var (certificate, intermediates) = _options.LoadNodeCertificate();
 
+			var certificateCommonName = certificate.GetNameInfo(X509NameType.SimpleName, false);
```

Comment:
@dhingrak looking around the code, I noticed we have a `GetCommonName()` method in `CertificateExtensions`.

just for uniformity reasons, i'd suggest we make this method public, changing it to `GetCommonName(this X509Certificate2 certificate)`, then we can call it here and in the client certificate authentication provider with `certificate.GetCommonName()`